Jenkinsfile

```diff
@@ -1,5 +1,17 @@
 #!groovy
 
+properties(
+    [
+        parameters(
+            [
+                booleanParam(name: 'ARCHIVE', defaultValue: false, description: 'Archive the build artifacts by pushing to an S3 bucket.'),
```

Comment:
Is this used anywhere in this repository? I know we have `CREATE_ARCHIVE`, but that's not related to pushing to s3; https://github.com/docker/containerd-packaging/blob/8c249eac8b8c9098ae42d1a6d828c1d8912cb702/Jenkinsfile#L52


(Couldn't find a reference to this parameter in our internal release repo either)